test/libsolidity/syntaxTests/events/event_ternary_operator.sol

```diff
@@ -0,0 +1,10 @@
+contract C {
+    event MyCustomEvent(uint);
+    function f() pure public {
+        true ? MyCustomEvent : MyCustomEvent;
```

Comment:
Is this breaking? I wouldn't be surprised if this pattern is seeing actual usage, unlike the `(a ? X.f : X.g).selector` business?

Reply:
It is, but it's not likely to be used much because it's mostly useless. You cannot use that with `emit` or store the value in a variable.

The most you can do seems to be this:
```
bytes32 s = (true ? MyCustomEvent : MyCustomEvent).selector;
```

which is similar to the case with functions.